stdlib/test/collections/test_dict.mojo

```diff
@@ -274,6 +274,40 @@ def test_dict_update_empty_new():
     assert_equal(len(orig), 2)
 
 
+@value
+struct DummyKey(KeyElement):
+    var value: Int
+
+    fn __hash__(self) -> Int:
+        return self.value
+
+    fn __eq__(self, other: DummyKey) -> Bool:
+        return self.value == other.value
+
+    fn __ne__(self, other: DummyKey) -> Bool:
+        return self.value != other.value
+
+
+def test_probing_error():
```

Comment:
can you rename this to `test_mojo_issue_1729`? that'll help point folks towards the issue context if/when someone trips this later